### ISSUE TITLE
Install Troubleshooting

### ISSUE THREAD

**Issue by Anachostic** [mikerodd/june-21#20]
Good day,

I am having an issue getting the june-21 plugin to load within my VST host.  Any guidance or troubleshooting steps would be appreciated.

My configuration:
- Windows 10 Pro 64-bit
- Cubase 5.5.3 64-bit
- ASIO4ALL 2.14

Attempts performed:
Placed all files into either of the VST folders (c:\Program Files\Steinberg\Cubase Studio 5\VSTPlugins or ...\Cubase Studio 5\vst3) either in the folder itself or a subfolder.
Each time I relocated the files, I either restarted Cubase or went to Devices>Plugin Information>Update to refresh plugins.  june-21 does not appear in the list on this screen, and is not available for selection from VST Instruments.

Note: The june-21.exe file does launch and generate sound external to Cubase.

Thank you for any suggestions for resolution.

**Comment by Anachostic** [comment]
I have figured out the solution through a long process of trial and error.  I'll document my findings for anyone else with a similar configuration.

There are 3 primary VST folders with the Cubase install:
c:\program files\Common Files\Steinberg\vst2
c:\program files\Steinberg\Cubase Studio 5\VSTPlugins
c:\program files\Steinberg\Cubase Studio 5\vst3

If you put all the files in vst2, june-21 doesn't show up anywhere.

If you put all the files in VSTPlugins, june-21 will show up in Devices>Plugin Information as "fx" and will not be selectable as an instrument.

If you put all the files in vst3, june-21 will show up as an instrument and will be selectable in VST Instruments, but will not initialize.

You must put csound64.dll somewhere in your system path.  I chose c:\windows\system32 despite that being the _wrong_ place to put shared library files.  After that, restarting Cubase should allow you to initialize june-21 and play the sounds.

To summarize my install success:  Put all files in c:\program files\Steinberg\Cubase Studio 5\vst3 (not VSTPlugins) and copy csound64.dll to a place in your system PATH.

Want to find a safer PATH for csound64.dll instead of the Windows folders?  Click Start and type Environment.  Click the search result for "Edit the system environment variables", then click the "Environment Variables" button in System Properties.  Double-click the row that says "Path".  Choose any of these folders to put the DLL in or add a new folder to the list for the DLL.
